src/client/common/utils/localize.ts

```diff
@@ -143,7 +142,7 @@ export namespace TensorBoard {
     export const launchNativeTensorBoardSessionCodeLens = l10n.t('▶ Launch TensorBoard Session');
     export const launchNativeTensorBoardSessionCodeAction = l10n.t('Launch TensorBoard session');
     export const missingSourceFile = l10n.t(
-        'We could not locate the requested source file on disk. Please manually specify the file.',
+        'We could not locate the requested source file on disk. You can manually specify the file.',
```

Comment:
```suggestion
        'Extension could not locate the requested source file on disk. Please manually specify the file.',
```

Restoring please here.

Reply:
@luabud How does this look?

Reply:
perfect, thank you!